tests/installation.bats

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bats
+
+# Run these tests from the repo root directory, for example
+# bats tests
+
+function setup {
+    export UNTAR_LOCATION=/tmp
+    export SOURCE_TARBALL_LOCATION=~/tmp/drupal_sprint_package.no_docker.$(cat .quicksprint_release.txt).tar.gz
+    export SPRINTDIR=~/sprint
+    # DRUD_NONINTERACTIVE causes ddev not to try to use sudo and add the hostname
+    export DRUD_NONINTERACTIVE=true
+    # Provide DHOST to figure out the docker host addr for curl
+    DHOST=127.0.0.1
+    if [ ! -z "$DOCKER_HOST" ]; then DHOST=$DOCKER_HOST; fi
+}
+
+# brew install jq p7zip bats-core composer
+# choco install -y jq 7zip composer
+# apt-get install jq p7zip-full
+# git clone git://github.com/bats-core/bats-core; cd bats-core; git checkout v1.1.0; sudo ./install.sh /usr/local
+# Passwordless sudo required.
+@test "check for prereqs (docker etc)" {
+    command -v curl
+    command -v jq
+    command -v 7z
+    command -v composer
+    # Make sure passwordless sudo is available
+    echo junk | sudo -S ls
+    docker run -t -v "$HOME:/tmp/home" -p 80:8088 busybox ls >/dev/null
+}
+
+
+@test "untar and run drupal_sprint_package" {
+    rm -rf "$UNTAR_LOCATION/drupal_sprint_package"
+    tar -C "$UNTAR_LOCATION" -zxf "$SOURCE_TARBALL_LOCATION"
+    chmod -R ugo+w $SPRINTDIR/ && rm -rf $SPRINTDIR/sprint-2*
```

Comment:
Probably doesn't exist before install_ddev.sh. I left this logic alone in the test because I'm not familiar with circle ci, but probably circle ci runs everything in a fresh environment so maybe this isn't necessary?

Reply:
Yeah, thanks. I'm definitely going to make that not fail, as it's optional.